### PR TITLE
League stats refactor

### PR DESCRIPTION
**Pull request by jaw772**
-Apply spec fixes

### CODE REVIEW

**Review by jacksonvaldez — APPROVED** [review]

File: lib/creator.rb [hunk 1]
```diff
@@ -1,24 +1,26 @@
 require_relative './tg_stat'
 require_relative './game'
 require_relative './team'
-# commented these out, spec still runs
+
 class Creator
-  attr_reader :teams_hash, :seasons_hash, :stats_hash, :games_hash
+  attr_reader :teams_hash, :seasons_hash, :stats_hash, :games_hash, :total_goals_by_team_hash
```
Comment (tjroeder) [inline lib/creator.rb:6]: Glad to see the inclusion of my refactor.

**Comment by Henchworm** [comment]
Good job getting the testing working!

File: lib/creator.rb [hunk 2]
```diff
@@ -62,4 +58,15 @@ def self.team_obj_creator(team_data, games_hash)
     end
     teams_hash
   end
+
```
Comment (tjroeder) [inline lib/creator.rb:61]: I like this hash refactor.

File: modules/league_stats.rb [hunk 3]
```diff
@@ -5,83 +5,43 @@ def count_of_teams
   end
 
   def best_offense
-    total_goals_by_team = {}
```
Comment (tjroeder) [inline modules/league_stats.rb:8]: This really DRY's up all the methods in LeagueStats module.

**Review by tjroeder — APPROVED** [review]
Approved changes.

File: modules/league_stats.rb (hunk 3 above)
Comment (Henchworm) [reply modules/league_stats.rb:8]: Much drier.

**Review by Henchworm — APPROVED** [review]

**Pull request merged**